security/tasks.py

```diff
@@ -490,6 +490,7 @@ def apply_async(self, args=None, kwargs=None, related_objects=None, **options):
         except (InterfaceError, OperationalError) as ex:
             logger.warn('Closing old database connections, following exception thrown: %s', str(ex))
             close_old_connections()
+            raise ex
```

Comment:
What's the reason?

Reply:
`apply_async` must return result or raise exception.